README.md

```diff
@@ -9,22 +9,41 @@ Rizin is portable and it can be used to analyze binaries, disassemble code,
 debug programs, as a forensic tool, as a scriptable command-line hexadecimal
 editor able to open disk files, and much more!
 
-To learn more on Rizin you may want to read the
-[official Rizin book](https://book.rizin.re).
+To learn more on Rizin, read the
+[official Rizin book](https://book.rizin.re). 
+
+# Table of Contents
```

Comment:
I am not sure if TOC makes sense for README, especially such a small one.

Reply:
this is easy to break. i'm against this.